l3kernel/l3doc.dtx

```diff
@@ -3741,6 +3753,13 @@ and all files in that bundle must be distributed together.
     \int_compare:nNnTF \@auxout = \@partaux
       { \@latexerr{\string\include\space cannot~be~nested}\@eha }
       { \@docinclude {#1} }
+    % check missing \endgroup, e.g., missing "\end{macro}" in time
+    \int_compare:nNnF { \tex_currentgrouplevel:D } = { 0 }
+      {
+        \int_compare:nNnT { \tex_interactionmode:D } = { 0 }
+          { \int_set:Nn \tex_interactionmode:D { 1 } }
+        \msg_fatal:nnn { l3doc } { missing-endgroup } { \currentfile }
+      }
```

Comment:
Expansion is missing, it should be `\msg_fatal:nne` since the msg functions stringify their argument.

Reply:
Thanks! I opened #1451 to sort this.